### ISSUE TITLE
rules_dotnet is incompatible with --incompatible_disallow_empty_glob

### ISSUE THREAD

**Issue by criemen** [bazel-contrib/rules_dotnet#458]
When trying to build on Bazel 8.0.0rc1, I'm getting
```
ERROR: Traceback (most recent call last):
        File "/private/var/tmp/_bazel_criemen/267ec2c545f0139b2caed9502387afe1/external/rules_dotnet++dotnet+dotnet_aarch64-apple-darwin/BUILD.bazel", line 33, column 16, in <toplevel>
                srcs = glob([
Error in glob: glob pattern 'sdk/**/AppHostTemplate/apphost.exe' didn't match anything, but allow_empty is set to False (the default value of allow_empty can be set with --incompatible_disallow_empty_glob).
```
See https://github.com/bazelbuild/bazel/issues/8195#issuecomment-2400850069 for how the pattern probably should look like instead.

**Comment by criemen** [comment]
This is with 0.15.1, but I don't see anything in the changelog for 0.16.0 that would address this.

**Comment by purkhusid** [comment]
This has been fixed on master and a new version 0.16.1 is in the queue in BCR. Thanks for reporting this!

**Comment by criemen** [comment]
Amazing, thanks for the quick turnaround on this!
